lib/protocol/message.proto

```diff
@@ -110,11 +110,14 @@ message DownloadProgressMessage {
 }
 
 message Folder {
-    string          id      = 1 [(gogoproto.customname) = "ID"];
-    string          label   = 2;
-    repeated Device devices = 3 [(gogoproto.nullable) = false];
-    uint32          flags   = 4;
-    repeated Option options = 5 [(gogoproto.nullable) = false];
+    string id                   = 1 [(gogoproto.customname) = "ID"];
+    string label                = 2;
+    bool   read_only            = 3;
```

Comment:
@AudriusButkevicius you recently made this a "folder type" enum internally, do we want to follow suite here?